packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -43,33 +44,46 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     [theme.palette.neutralLighterAlt]
   );
 
+  const [showPeople, setShowPeople] = useState(false);
+
   return (
-    <Stack verticalFill horizontalAlign="stretch" className={containerClassName} data-ui-id={props.dataUiId}>
-      <Stack.Item styles={notificationsContainerStyles}>
-        <Stack>
-          <ComplianceBanner {...props.complianceBannerProps} />
-        </Stack>
-        {props.errorBarProps !== false && (
+    <Stack className={mergeStyles({ width: '100%', height: '100%' })}>
+      <ParticipantPane hidden={!showPeople} closePane={() => setShowPeople(false)} />
+      <Stack
+        verticalFill
+        horizontalAlign="stretch"
+        className={showPeople ? mergeStyles(containerClassName, { display: 'none' }) : containerClassName}
+        data-ui-id={props.dataUiId}
+      >
+        <Stack.Item styles={notificationsContainerStyles}>
           <Stack>
-            <ErrorBar {...props.errorBarProps} />
-          </Stack>
-        )}
-        {!!props.mutedNotificationProps && <MutedNotification {...props.mutedNotificationProps} />}
-      </Stack.Item>
-
-      <Stack.Item styles={callGalleryStyles} grow>
-        {props.onRenderGalleryContent && (
-          <Stack verticalFill styles={mediaGalleryContainerStyles}>
-            {props.onRenderGalleryContent()}
+            <ComplianceBanner {...props.complianceBannerProps} />
           </Stack>
-        )}
-      </Stack.Item>
+          {props.errorBarProps !== false && (
+            <Stack>
+              <ErrorBar {...props.errorBarProps} />
+            </Stack>
+          )}
+          {!!props.mutedNotificationProps && <MutedNotification {...props.mutedNotificationProps} />}
+        </Stack.Item>
 
-      {props.callControlProps?.options !== false && (
-        <Stack.Item className={callControlsContainerStyles}>
-          <CallControls {...props.callControlProps} />
+        <Stack.Item styles={callGalleryStyles} grow>
+          {props.onRenderGalleryContent && (
+            <Stack verticalFill styles={mediaGalleryContainerStyles}>
+              {props.onRenderGalleryContent()}
+            </Stack>
+          )}
         </Stack.Item>
-      )}
+
+        {props.callControlProps?.options !== false && (
```

Comment:
[feel free to disregard]
might be good to define a new variable to say why we are showing the callControls or not.